specification/resources/resource-manager/Microsoft.Resources/preview/2022-08-01-preview/deploymentStacks.json

```diff
@@ -0,0 +1,1353 @@
+          },
+          {
+            "$ref": "#/parameters/DeploymentStackNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the Deployment Stack.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentStack"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentStacksError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DeploymentStacks"
+        ],
+        "operationId": "DeploymentStacks_DeleteAtResourceGroup",
+        "description": "Deletes a Deployment Stack by name. When operation completes, status code 200 returned without content.",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "DeploymentStacksDelete": {
+            "$ref": "./examples/DeploymentStackResourceGroupDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DeploymentStackNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DeleteResourceParameter"
+          },
+          {
+            "$ref": "#/parameters/DeleteResourceGroupParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Deployment Stack deleted."
+          },
+          "202": {
+            "description": "Accepted - Check location header for deletion status.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Deployment Stack does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentStacksError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Resources/deploymentStacks/{deploymentStackName}": {
```

Comment:
>"/subscriptions/{subscriptionId}/providers/Microsoft.Resources/deploymentStacks/{deploymentStackName}" [](http://example.com/codeflow?start=4&length=102)

Just curious:

Can a deploymentStack "stack1" created via:

PUT "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Resources/deploymentStacks/stack1"

be accessed via:

GET "/subscriptions/{subscriptionId}/providers/Microsoft.Resources/deploymentStacks/stack1"
?

Reply:
It cannot be accessed via that GET because stack1 exists in the resource group {resourceGroupName}.

Reply:
Generally, my understanding has been that any resource which exists under resource group scope, is automatically visible under parent (i.e., subscription here) scope.